test/test_utils.py

```diff
@@ -83,6 +85,34 @@ def build_pairs(with_preprocessor=False):
     return Dataset(X[c], target, None, X[c[:, 0]])
 
 
+def build_triplets(with_preprocessor=False):
+  input_data, labels = load_iris(return_X_y=True)
+  X, y = shuffle(input_data, labels, random_state=SEED)
+  constraints = Constraints(y)
+  triplets = constraints.generate_knntriplets(X, 3, 10)
```

Comment:
maybe 3 or 4 impostors is sufficient? to make the tests faster

Reply:
Yes, I think you are right! Done.